index.html

```diff
@@ -2891,6 +2896,66 @@ <h3 id="applying">
         </ol>
       </section>
     </section>
+      <section>
```

Comment:
This section should be part of "8. Manifest and its members". Not it's own section.